### ISSUE TITLE
Subscribe to all tabs of the browser

### ISSUE THREAD

**Issue by Sa1ch** [morozovsk/workerman-examples#3]
Доброго дня! Подскажите не знающему человеку пожалуйста.
Как поддерживать активным соединение если открыто много вкладок браузера?
Из первого примера: если открыть вторую вкладку то все сообщения перестанут доходить до первой вкладки, а будут приходить во вторую.
Из второго примера: в приципе работает, но если человек открыл много вкладок? Например 10?

В принципе можно попробовать при подключении к сокету передать гет параметр с префикосом (номером соединения). Но мне кажется это топорно и не правильно. Есть какое-то решение?

**Comment by morozovsk** [comment]
в обоих примерах user_id и соответствующее ему соединение connection_id сохраняются в массив $users:
['user_id' => 'connection_id']
что мешает для одного user_id сохранять несколько коннектов, или для connection_id хранить user_id?

**Comment by maximal** [comment]
Верно. Для каждого пользователя хранить массив соединений, а не только одно соединение (первое или последнее).
